bottlerocket/samples/Makefile.toml

```diff
@@ -0,0 +1,242 @@
+[config]
+skip_core_tasks = true
+
+[env]
+AGENT_IMAGE_REGISTRY = { value = "public.ecr.aws/bottlerocket-test-system", condition = { env_not_set = ["AGENT_IMAGE_REGISTRY"] } }
+AGENTS = ["eks-resource-agent"]
+METADATA_BASE_URL = { value = "https://updates.bottlerocket.aws/2020-07-07", condition = { env_not_set = ["METADATA_BASE_URL"] } }
+ARCHITECTURE = { value = "x86_64", condition = { env_not_set = ["ARCHITECTURE"] } }
+CLUSTER_TYPE = { value = "eks", condition = { env_not_set = ["CLUSTER_TYPE"] } }
+ASSUME_ROLE = { value = "~", condition = { env_not_set = ["ASSUME_ROLE"] } }
+AWS_REGION = { value = "us-west-2", condition = { env_not_set = ["AWS_REGION"] } }
```

Comment:
Is it ok for these to be unprefixed? i.e. in Bottlerocket we use `BUILDSYS_` as a prefix for env vars. Will these clash with anything?

Reply:
They won't be exported outside of the makefile process, so the only risk is that they were defined prior to running `cargo make` and contain values that will clash with the desired values for the generated sample. That was more of a discussion for the previous PRs though, where I added the sample files, so I'm hesitant to change it now. Either way, they won't have any side effects outside of the `cargo make` invocation.